python/grass/jupyter/timeseriesmap.py

```diff
@@ -101,32 +101,15 @@ def collect_layers(timeseries, element_type, fill_gaps):
     return names, dates
 
 
-class MethodCallCollector:
-    """Records lists of GRASS modules calls to hand to Map.run().
-
-    Used for base layers and overlays in TimeSeriesMap visualizations."""
-
-    def __init__(self):
-        """Create list of GRASS display module calls"""
-        self.calls = []
-
-    def __getattr__(self, name):
-        """Parse attribute to GRASS display module. Attribute should be in
-        the form 'd_module_name'. For example, 'd.rast' is called with 'd_rast'.
-        """
-        # Check to make sure format is correct
-        if not name.startswith("d_"):
-            raise AttributeError(_("Module must begin with 'd_'"))
-        # Reformat string
-        grass_module = name.replace("_", ".")
-        # Assert module exists
-        if not shutil.which(grass_module):
-            raise AttributeError(_("Cannot find GRASS module {}").format(grass_module))
-
-        def wrapper(**kwargs):
-            self.calls.append((grass_module, kwargs))
-
-        return wrapper
+def check_timeseries_exists(timeseries):
```

Comment:
It could take name and element_type and return boolean. The calling function could then give a message with the right type mentioned.

Alternatively, the message can be `temporal dataset {name} of type {element_type}` or something like that.